src/Dfc.CourseDirectory.WebV2/Filters/AuthorizeAdminAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Dfc.CourseDirectory.WebV2.Security;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Dfc.CourseDirectory.WebV2.Filters
+{
+    public class AuthorizeAdminAttribute : ActionFilterAttribute
```

Comment:
Why not derivative of `IAuthorizationFIlter`?

Reply:
Relies on `NotAuthorizedException` being caught and handled by an exception filter. Exception filters don't catch errors from `IAuthorizationFilter`s, it seems.

Reply:
Why does it need to throw a `NotAuthorizedException`? Can it not just return the response from the filter?

Reply:
It could but then we have the logic in two places instead of one.